### ISSUE TITLE
Loading existing data from database

### ISSUE THREAD

**Issue by nutullahkisa** [OrchardCMS/Orchard#8138]
Please note: I need to find a solution for this issue by Monday so any feedback/advice that you may have is appreciated.

I have created a basic module to display the contact list for employees. When I add new employees through the module, they update on the database and display in the module correctly; however, the employees that already exist in the database (they existed in our old database and I imported them into the newly created module database) do not display in the module. There are no differences between the columns for the employees that I add manually through the module or the ones that I imported in, but only the ones that I add in manually display.

In the migration file it looks like the employees are being added to an additional table when I create new employees manually through the module.

How can I make sure the employees that we already have can be imported in properly to the correct tables so that they can be displayed in the module? We have a large database of employees so it's not an option to manually enter each one.

Thank you

**Comment by sebastienros** [comment]
Are these content items, as in you created a Part for them ? In this case you would also need to create a record in the ContentItemVersion and ContentItemRecord. Another solution is to create an import file in XML. There are several examplesnon how to do it.

**Comment by nutullahkisa** [comment]
Thanks for your response. They are content items - I created a record in the ContentItemVersion and ContentItemRecord for then.

Can you point me to where I can find the examples on this? Thanks again

**Comment by ViRuSTriNiTy** [comment]
You could also [enable logging](https://weblogs.asp.net/bleroy/logging-sql-queries-in-orchard) to determine where user queries go wrong by copy and pasting the user queries into a SQL debugger. For sure you missed something, a flag, a record, etc.

**Comment by nutullahkisa** [comment]
Thank you for your responses. 

When I checked the log file it was empty.

I was able to do this by creating the import file and implementing the importing and exporting methods. I used the User module as a model. 
I created one dummy object and then exported it to XML and then modeled my data off of that for the import file. I had to add handling for null values because if I imported a field as null that is stored as an int then the entire row would store as blank.
